src/function/string/operations/include/left_operation.h

```diff
@@ -18,8 +18,7 @@ struct Left {
 public:
     static inline void operation(
         ku_string_t& left, int64_t& right, ku_string_t& result, ValueVector& resultValueVector) {
-        auto len = right > 0 ? min(left.len, (uint32_t)right) :
-                               max(left.len + (uint32_t)right, (uint32_t)0u);
+        auto len = right > 0 ? min(left.len, (uint32_t)right) : 0u;
```

Comment:
I think left/right operation can take negtaive index.
eg. 
```
D select left('123', -1) from test;
┌─────────────────┐
│ left('123', -1) │
├─────────────────┤
│ 12              │
└─────────────────┘
```
Also change code for right as well